packages/react-components/src/templates/__tests__/DiscoverTabs.test.tsx

```diff
@@ -0,0 +1,72 @@
+import { createNewsResponse } from '@asap-hub/fixtures';
+import { render, screen } from '@testing-library/react';
+
+import DiscoverTabs from '../DiscoverTabs';
+
+describe('Renders the training tab', () => {
+  it('renders the tutorials title and subtitle', () => {
```

Comment:
These tests feel duplicated. You should be testing that it displays a heading with supplied text and that it can render a list of news cards